react/Table/Virtualized/index.jsx

```diff
@@ -43,10 +47,9 @@ const VirtualizedTable = forwardRef(
     }
 
     const handleSort = property => {
-      const isAsc = orderBy === property && order === 'asc'
+      const isAsc = order.attribute === property && order.order === 'asc'
```

Comment:
there is no conditional chaining `?.` here, is it working if `attribute` or `order` are undefined ?